analytics/app/src/main/java/com/google/firebase/quickstart/analytics/java/MainActivity.java

```diff
@@ -104,28 +106,31 @@ protected void onCreate(Bundle savedInstanceState) {
             setUserFavoriteFood(userFavoriteFood);
         }
 
-
         // Create the adapter that will return a fragment for each image.
-        mImagePagerAdapter = new ImagePagerAdapter(getSupportFragmentManager(), IMAGE_INFOS);
+        mImagePagerAdapter = new ImagePagerAdapter(getSupportFragmentManager(), IMAGE_INFOS, getLifecycle());
```

Comment:
I think `getParentFragmentManager()` is [preferred in fragments](https://developer.android.com/guide/fragments/fragmentmanager#access).

```suggestion
        mImagePagerAdapter = new ImagePagerAdapter(getParentFragmentManager(), IMAGE_INFOS, getLifecycle());
```

Reply:
I just noticed this is being called in an Activity and not a Fragment. So this comment may be disregarded. My bad

Reply:
No issues.